### PR TITLE
latency improvements for submitting user operations

### PR DESCRIPTION
**Pull request by alvrs**

### CODE REVIEW

File: packages/account-kit/src/steps/finalizing/FinalizingStep.tsx [hunk 1]
```diff
@@ -32,11 +33,18 @@ export function FinalizingStep() {
       if (!appAccountClient) throw new Error("App account client not ready.");
       if (!registerDelegationSignature) throw new Error("No delegation signature.");
 
+      // TODO: should this use `callWithSignature`?
```
Comment (holic) [inline packages/account-kit/src/steps/finalizing/FinalizingStep.tsx:36]: It does a few lines below?

File: packages/common/package.json [hunk 2]
```diff
@@ -63,6 +63,7 @@
     "execa": "^7.0.0",
     "p-queue": "^7.4.1",
     "p-retry": "^5.1.2",
+    "permissionless": "^0.1.17",
```
Comment (holic) [inline packages/common/package.json:66]: I don’t think this is used?

File: packages/account-kit/src/steps/finalizing/FinalizingStep.tsx (hunk 1 above)
Comment (alvrs) [reply packages/account-kit/src/steps/finalizing/FinalizingStep.tsx:36]: yeah i meant the `callWithSignature` util we have (that i believe we use for `registerSpender`)

File: packages/common/package.json (hunk 2 above)
Comment (alvrs) [reply packages/common/package.json:66]: oh yeah you're right, removed all changes to `common` that had to do with smart accounts again

File: packages/account-kit/src/steps/finalizing/FinalizingStep.tsx (hunk 1 above)
Comment (holic) [reply packages/account-kit/src/steps/finalizing/FinalizingStep.tsx:36]: ohhh I see! that function is built to request the signature so wasn't immediately available for use here could be adapted

**Pull request merged**